sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventDataBatch.java

```diff
@@ -55,18 +55,28 @@
      *
      * @return The number of {@link EventData events} in the batch.
      */
-    int getSize() {
+    public int getCount() {
```

Comment:
Now that this is public API, can I register my preference for size over count? :-) It is pretty standard in Java to use 'size'.

Reply:
Do we have getSize (number of items in batch) and GetSizeInBytes?